assemblies/features/base/src/main/filtered-resources/resources/bin/karaf

```diff
@@ -322,6 +322,7 @@ run() {
                     --add-exports=jdk.naming.rmi/com.sun.jndi.url.rmi=ALL-UNNAMED \
                     --add-exports=java.rmi/sun.rmi.registry=ALL-UNNAMED \
                     --add-exports=java.security.sasl/com.sun.security.sasl=ALL-UNNAMED \
+                    --add-exports=java.naming/com.sun.jndi.ldap=ALL-UNNAMED \
```

Comment:
Is it valid for any JDK ?

Reply:
This is related on java 9 above

Reply:
my point is not about JDK version, it's more the JDK provider, between Adoptium/OpenJDK/Oracle/IBM. As it's command line argument, it's OK, just wondering ;)

Reply:
Oh, misunderstood. OpenJDK is JDK provider. Background: upgrade from java11 to java17 created an issue with camel-ldap -component in karaf container. Found out that com.sun.jndi.ldap was not exported by default.

Reply:
OK, fair enough. Thanks.